clustered/server/src/test/java/org/ehcache/clustered/server/store/ClusterTierActiveEntityTest.java

```diff
@@ -238,22 +235,23 @@ public void testAppendInvalidationAcksTakenIntoAccount() throws Exception {
     activeEntity.connected(context2.getClientDescriptor());
     activeEntity.connected(context3.getClientDescriptor());
 
-    ServerStoreMessageFactory messageFactory = new ServerStoreMessageFactory();
+    UUID client2Id = UUID.randomUUID();
```

Comment:
I don't see these UUID in used. Bad rebase?

Reply:
Looks like it. Fixed.